fastlane/Fastfile

```diff
@@ -259,11 +264,6 @@ platform :android do
   end
 
   # Authenticator
-  desc "Runs tests"
-  lane :checkAuthenticator do
-    gradle(tasks: ["authenticator:testDebug", "authenticator:lintDebug", "authenticator:detekt","authenticator:koverXmlReportDebug"])
-  end
```

Comment:
🗒️ This is no longer needed since we're consolidating the checks. Both `build.yml` workflows will trigger the same `check` lane.